### PR TITLE
Adding Production values to vSphere platform

### PR DESCRIPTION
**Pull request by vavuthu**
Adding Production values to vSphere platform

Also, added separate production yaml file for ocs4qe lab which has
hardware limitations.

Signed-off-by: vavuthu <vavuthu@redhat.com>

Fixes: #1001

### CODE REVIEW

File: conf/ocsci/production-vsphere-upi-ocs4qe-lab.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+# This config will provide the 12 vCPU's to VM.
+# We are restricting to 12 due to hardware limitation in the lab
+DEPLOYMENT:
+  allow_lower_instance_requirements: true
+ENV_DATA:
+  worker_num_cpus: '12'
+  master_num_cpus: '4'
```
Comment (petr-balogh) [inline conf/ocsci/production-vsphere-upi-ocs4qe-lab.yaml]: Do we have enough memory to increase on our lab env or not really?

**Comment by petr-balogh** [comment]
@clacroix12 FYI for our lab we will have to use lab production config as we have lack of resources in our lab

File: conf/ocsci/production-vsphere-upi-ocs4qe-lab.yaml (hunk 1 above)
Comment (vavuthu) [reply conf/ocsci/production-vsphere-upi-ocs4qe-lab.yaml]: We cannot accommodate 64GB

File: conf/ocsci/production-vsphere-upi-ocs4qe-lab.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+---
```
Comment (clacroix12) [inline conf/ocsci/production-vsphere-upi-ocs4qe-lab.yaml]: If these are not true production values should we remove `production-` from the filename?

Comment (vavuthu) [reply conf/ocsci/production-vsphere-upi-ocs4qe-lab.yaml]: Done

**Pull request merged**